apps/src/codegen.js

```diff
@@ -253,14 +274,25 @@ exports.marshalNativeToInterpreter = function (interpreter, nativeVar, nativePar
       nativeParentObj: nativeParentObj,
     };
     for (i = 0; i < exports.asyncFunctionList.length; i++) {
-      // If this is on our list of "custom marshal" objects - or if it a property
-      // on one of those objects (other than a function), create a special
-      // "custom marshal" interpreter object to represent it
+      // Mark if this should be nativeIsAsync:
       if (nativeVar === exports.asyncFunctionList[i]) {
         makeNativeOpts.nativeIsAsync = true;
         break;
       }
     }
+    for (i = 0; i < exports.nativeCallsInterpreterFunctionList.length; i++) {
+      // Mark if this should be nativeCallsBackInterpreter:
+      if (nativeVar === exports.nativeCallsInterpreterFunctionList[i]) {
```

Comment:
This is just a search; it makes me wish we were using something like [`_.includes`](https://lodash.com/docs#includes) instead, though we seem to avoid helper libraries in this file.